entity-framework/core/miscellaneous/cli/powershell.md

```diff
@@ -9,7 +9,7 @@ uid: core/miscellaneous/cli/powershell
 
 The Package Manager Console (PMC) tools for Entity Framework Core perform design-time development tasks. For example, they create [migrations](/aspnet/core/data/ef-mvc/migrations?view=aspnetcore-2.0), apply migrations, and generate code for a model based on an existing database. The commands run inside of Visual Studio using the [Package Manager Console](/nuget/tools/package-manager-console). These tools work with both .NET Framework and .NET Core projects.
 
-If you aren't using Visual Studio, we recommend the [EF Core Command-line Tools](dotnet.md) instead. The CLI tools are cross-platform and run inside a command prompt.
+If you aren't using Visual Studio, we recommend the [EF Core Command-line Tools](dotnet.md) instead. The .NET Core CLI tools are cross-platform and run inside a command prompt and work with .NET Core projects only.
```

Comment:
@bricelam I'd be grateful if you would particularly check the last phrase I added. It's meant to be "as opposed to what we said line 10" where we work with both .NET Framework and .NET Core projects but I'm happy to change it if I've misunderstood what you were trying to stress here.

Reply:
Hmm, I think it just adds confusion. Also, `dotnet ef` works on .NET Framework projects (SDK-style only).

We added the one on line 10 because in the beginning, a lot of users didn't know they also worked with .NET Core projects. There was a bit of misinformation somewhere that you *had* to use `dotnet ef` on .NET Core projects.

Reply:
Fair enough. I'll take out that phrase and check in. At that point our docs will be consistent and say ".NET Core CLI" everywhere. If there's a bigger change to change ".NET Core CLI" to ".NET CLI" then that will affect a lot more docs - so let's do that separately.